package.json

```diff
@@ -73,15 +95,17 @@
     "preprocess": "run-p preprocess:*",
     "preprocess:svg-to-twig": "gulp svg-to-twig",
     "preprocess:theo-to-mdx": "gulp theo-to-mdx",
-    "watch-preprocess": "gulp watch",
+    "watch-preprocess": "gulp watch-preprocess",
+    "build": "gulp build",
     "check-lint": "run-s check-lint:*",
-    "check-lint:prettier": "prettier --ignore-path .gitignore --list-different '**/*.{scss,js,mdx}'",
+    "check-lint:prettier": "prettier --ignore-path .gitignore --check '**/*.{scss,js,mdx}'",
     "check-lint:css": "stylelint --ignore-path .gitignore '**/*.scss'",
     "check-lint:js": "eslint --ignore-path .gitignore '**/*.{js,mdx}'",
     "lint": "run-s lint:*",
     "lint:prettier": "prettier --ignore-path .gitignore --write '**/*.{scss,js,mdx}'",
     "lint:css": "stylelint --ignore-path .gitignore --fix '**/*.scss'",
-    "lint:js": "eslint --ignore-path .gitignore --fix '**/*.{js,mdx}'"
+    "lint:js": "eslint --ignore-path .gitignore --fix '**/*.{js,mdx}'",
+    "prepublishOnly": "npm ci && npm run check-lint && npm run preprocess && npm run build"
```

Comment:
Could you add a CONTRIBUTING.md file with instructions on how to publish to npm? (even if it's just "run `npm version` and `npm publish`)

Reply:
Good idea, added ✅